.github/workflows/deployDemo.yml

```diff
@@ -27,18 +27,30 @@ jobs:
         uses: docker/setup-qemu-action@v1
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
+      - uses: azure/login@v1
```

Comment:
Note: all of the deploy*.yml files have identical changes (literally, I stamped them out with `patch`).